CMakeLists.txt

```diff
@@ -38,6 +42,18 @@ option(RAY_USE_GLOG
   "Build the logging system using glog"
   ON)
 
+if (RAY_BUILD_PYTHON)
+  set(CMAKE_RAY_LANG_PYTHON YES)
+endif ()
+
+if (RAY_BUILD_JAVA)
+  set(CMAKE_RAY_LANG_JAVA YES)
+endif ()
+
+if ("${CMAKE_RAY_LANG_JAVA}" STREQUAL "NO" AND "${CMAKE_RAY_LANG_PYTHON}" STREQUAL "NO")
+  message(FATAL_ERROR "NO RAY LANG DEFINED")
```

Comment:
"Please specify which language support you want to build by passing -DRAY_BUILD_PYTHON=on and/or -DRAY_BUILD_JAVA=on to CMake"

Reply:
OK!